Rmarkdown/Rmarkdown-intro.Rmd

```diff
@@ -0,0 +1,643 @@
+If the file is in a sub-directory, use `here::here()` to form paths relative to the project root.
+
+## Inline code
+
+In addition to the output from chunks of code, you can insert code directly into text.
+Inline code is enclosed by back-ticks and starts with an `r`.
+
+Seven times six is `` `r '\x60r 7 * 6\x60'` ``
+
+Seven times six is `r 7 * 6`
+
+If you want numbers written as words, use the package `english`.
+
+Seven times six is `` `r 'english::words(\x60r 7 * 6\x60)'` ``
+
+Seven times six is `r english::words(7 * 6)`
+
+It is best to keep inline code short to keep the text readable.
+One trick is to do all necessary calculation in a previous chunk, so only the name of the object with the result needs to be in the inline code.
+If there are many results to report, consider storing them in a list.
+
+
+# Figures, tables and equations
+
+## Figures made in R
+
+Plots can be included with an chunk that makes a figure with either base plot or ggplot.
+
+If you make the plot with ggplot, remember to print it
+
+```` 
+```{r histogram,  fig.cap = "An embedded figure"}`r ''`
+p <- ggplot(penguins, aes(x = bill_length_mm)) + 
+  geom_histogram()
+p # remember to print the plot
+```
+````
+
+### Figure chunk options
+
+There are several useful chunk options for figures, including
+
+-   `fig.cap` figure caption.
+-   `fig.height` figure height *in inches* (1 inch = 25.4 mm)
+-   `fig.width` figure width *in inches*
+
+## Embedding external images
+
+Photographs and other figures that have been prepared outside of R can be included with a link.
+
+`![Alt description](image.jpg)`
+
+But you have much more control with `knitr::include_graphics()`
+
+```` 
+```{r include-figure, out.width = "100%", fig.cap = "An embedded figure"}`r ''`
+knitr::include_graphics("my_image.png")
+```
+````
+
+Use the `out.width` and `out.height` chunk options to set the size of the figure.
+
+## Tables
+
+You can make tables in markdown by hand, but it is so much easier to use R.
+
+Simple tables can be made with the function `knitr::kable`.
+Several packages, including `kableExtra` and [`gt`](https://gt.rstudio.com/articles/intro-creating-gt-tables.html) can make beautiful tables
+
+### kable
+
+```{r kable}
+
+knitr::kable(x = slice(penguins, 1:5), #the data for the table
+             caption = "The top penguins" # the caption
+)
+```
+
+### kableExtra
+
+`kableExtra` extends `knitr::kable`, letting you add extra rows of headers, repeating headers if the table spans multiple pages and many other features.
+
+```{r, message = FALSE}
+library(kableExtra)
+penguins %>% 
+  #rename some columns
+  rename(length_mm = bill_length_mm, depth_mm = bill_depth_mm) %>% 
+  #get first five rows
+  slice(1:5) %>% 
+kbl(booktabs = TRUE, caption = "The top penguins") %>% 
+  kable_styling(full_width = TRUE) %>%
+  add_header_above(c(" ", " ", "bill" = 2, "body" = 2, " ", " "))
+  
+```
```

Comment:
Could add the code to make a table landscape (wide table)

Reply:
That is a latex package to addon - now link to the solution